data/remnant/remnant jobs.txt

```diff
@@ -280,14 +281,15 @@ mission "Remnant: Logistics Industrial Delivery"
 		dialog phrase "generic cargo on visit"
 	on complete
 		dialog `A project technician inspects the <cargo> and approves your payment of <payment>.`
-		payment 20000 25
+		payment
+		payment 10000
```

Comment:
For a 100 cargo at 1 jump this would originally pay 25000, and should now pay 40000.